git add .

```diff
@@ -0,0 +1,14 @@
+identifier: JSON
```

Comment:
Please remove this file ;)

Reply:
Oops, I thought it wasn't there since git status doesn't show this file